tools/mitre/mitredb.py

```diff
@@ -344,6 +433,31 @@ def parse_table_(function, data_object):
         if const.DEPRECATED_j in data_object:
             table.deprecated = data_object[const.DEPRECATED_j]
 
+    # Alias
+    if data_object.get(const.ALIAS_j):
```

Comment:
Create a new function with all this code to not repeat it twice

Reply:
Done